Examples/Io/Csv/src/CsvOutputData.hpp

```diff
@@ -349,4 +349,19 @@ struct ProtoTrackData {
   DFE_NAMEDTUPLE(ProtoTrackData, trackId, measurementId, x, y, z);
 };
 
+struct BucketData {
```

Comment:
That name seems a bit generic - could you tune it a bit an provide a small explanation?